Docs: Add Visualizations and Widgets Split to what's new 10.1 (#72009)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v10-1.md

{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-disconnect-values-examples.png" max-width="750px" caption="Disconnect values in time series, trend, and state timeline visualizations" >}}

> ### Visualizations and Widgets Split

Reviewer: ```suggestion
### Visualizations and Widgets split
```

<!--Alexa Vargas -->

> _Available to some of our free tier users of Grafana, this feature is experimental._

Reviewer: ```suggestion
_Experimental in some of our free tiers of Grafana._
```

> This experimental feature brings a clear distinction between panel plugins: visualizations and widgets. Now, easily add widgets like Text, News, and Annotation List without selecting a datasource. The plugins list and library panels are filtered based on widget or visualization type, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our cloud free tier users.

Reviewer: ```suggestion
This experimental feature brings a clear distinction between two panel plugin types: visualizations and widgets. Now, easily add widgets like Text, News, and Annotation list, without selecting a data source. The plugins list and library panels are filtered based on widget or visualization type, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our Cloud free tier users.
```

{{< video-embed src="/media/docs/grafana/panels-visualizations/screen-recording-grafana-10.1-flamegraph-whatsnew.mp4" >}}

### Visualizations and Widgets split

<!-- Alexa Vargas -->

> _Experimental in some of our free tiers of Grafana._

Reviewer 2: Is this how we communicate it? I'd expect to follow the same as any other release type "_Experimental in all editions of Grafana._" In the end it's available for everyone but they'd have to enable the feature toggle.

Reviewer: I'm wondering the same here. As well as whether we should add something about feature toggles.

> This experimental feature brings a clear distinction between two panel plugin types: visualizations and widgets. Now, easily add widgets like Text, News, and Annotation list, without selecting a data source. The plugins list and library panels are filtered based on widget or visualization type, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our Cloud free tier users.

Author: @imatwawana I am not sure about this part, can you help me? 🙈 

>This experimental feature brings a clear distinction between two panel plugin types: visualizations and widgets. Now, easily add widgets like Text, News, and Annotation list, without selecting a data source.

In our current system, we don't have the concept of "widgets". We have different plugin types (Timeseries, Text, News, Stat panel), and some of them use data sources, while others don't need one. 
So, we're introducing a new concept called "widgets." Widgets are panels that don't require a data source. For example, we have widgets like the Text panel, News panel, and others.

I am wondering if this will bring more clarity:

```suggestion
This experimental feature introduces a clear distinction between two different categories of panel plugin types: visualizations panels that consume a data source and a new type called "widgets" that don't require a data source. Now, users can easily add widgets like Text, News, and Annotation list without the need to select a data source. The plugins list and library panels are filtered based on widget or visualization type, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our Cloud free tier users.
```

Also, I want to add this screenshot, do you how do I do that? 

![WidgetVizSplit](https://github.com/grafana/grafana/assets/239999/0c5cb139-04f1-4ffd-a584-787b73f3567c)

Reviewer: Yes that's much clearer. A couple tweaks:
```suggestion
This experimental feature introduces a clear distinction between two different categories of panel plugin types: visualization panels that consume a data source and a new type called _widgets_ that don't require a data source. 

Now, you can easily add widgets like Text, News, and Annotation list without the need to select a data source. The plugins list and library panels are filtered based on whether you've selected a widget or visualization, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our Cloud free tier users.
```

Reviewer: Upload your image [here](https://admin.grafana.com/upload/media/docs/grafana/dashboards/) and then add in the reference to the image location starting the file path at "media". If you want to control the width of the image, then use the [`figure` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#figure-shortcode) instead of regular markdown to add the image. The `figure` shortcode has a max-width property that you can set.

> This experimental feature brings a clear distinction between two panel plugin types: visualizations and widgets. Now, easily add widgets like Text, News, and Annotation list, without selecting a data source. The plugins list and library panels are filtered based on widget or visualization type, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our Cloud free tier users.

Reviewer 2: Similar here "Early access to this exciting update will be granted to some of our Cloud free tier users". We usually say something like "To try X, enable the `Y` feature toggle."

Author: I applied the suggestions :)

{{< video-embed src="/media/docs/grafana/panels-visualizations/screen-recording-grafana-10.1-flamegraph-whatsnew.mp4" >}}

### Visualizations and Widgets split

<!-- Alexa Vargas -->

_Experimental in all editions of Grafana._

This experimental feature introduces a clear distinction between two different categories of panel plugin types: visualization panels that consume a data source and a new type called _widgets_ that don't require a data source. 

> Now, you can easily add widgets like Text, News, and Annotation list without the need to select a data source. The plugins list and library panels are filtered based on whether you've selected a widget or visualization, providing a streamlined editing experience. If you want to try this new feature, you can enable the feature flag, `vizAndWidgetSplit`.

Reviewer: Adding enablement info for Cloud and using standard language for the feature toggle

```suggestion
Now, you can easily add widgets like Text, News, and Annotation list without the need to select a data source. The plugins list and library panels are filtered based on whether you've selected a widget or visualization, providing a streamlined editing experience. 

To try out the visualizations and widgets split, enable the `vizAndWidgetSplit` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
```

Author: @imatwawana I added the suggestion manually, I was resolving a conflict and could not add the commit from here. Can you take a last look? 🙈

Reviewer: Looks good!
